sources/api/openapi.yaml

```
@@ -314,3 +314,13 @@ paths:
                 $ref: "ConfigurationFiles"
         500:
           description: "Server error"
+
+  /actions/reboot:
+    post:
+      summary: "Reboot"
+      operationId: "reboot"
+      responses:
+        204:
+          description: "Successfully requested - reboot in 10 seconds"
```

```suggestion
          description: "Reboot requested"
```
Let's not include the time here; the time could change and I think there's no need to remember to include it here.